### PR TITLE
Add ccm workload aware recommendations to documentation

### PR DESCRIPTION
**Pull request by JackHFinkel**
### What does this PR do? What is the motivation?

The goal of this PR is to add released workload aware ccm recommendations to recommendations docs 
[Here is our google doc we made a draft of the documented recs for](https://docs.google.com/document/d/1mMUvVwj-qHdnZEFUiy1tAAobvNpGuteVMvaRCseN6iY/edit?tab=t.0)

Local validation
image
image

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

**Review by dave-handy — APPROVED** [review]

File: content/en/cloud_cost_management/recommendations/_index.md [hunk 1]
```diff
@@ -75,24 +75,32 @@ For each cloud account that you would like to receive recommendations for:
...
+| Unused resource | AWS | DynamoDB | Unused DynamoDB | A DynamoDB table has 0 consumed reads and 0 consumed non-replica writes. |
 | Unused resource | AWS | DynamoDB | Unused DynamoDB Global Secondary Index | A DynamoDB table's Global Secondary Index (GSI) has 0 consumed reads. |
 | Unused resource | AWS | Autoscaling groups (ASG) | ASGs with legacy instance types | An autoscaling group that includes legacy instance types. | [Datadog Agent][5] |
 | Unused resource | AWS | DynamoDB | DynamoDB Delete Extra On-Demand Backups | A DynamoDB table has charges for more than 2 on-demand backups. | |
 | Previous generation resource | AWS | EC2 | Legacy EC2 Instance | EC2 Instances that are previous generation, and can be upgraded to a newer instance type. | [Datadog Agent][5] |
+| Unused resource | AWS | EC2 | Unused EC2 instance running Redis | EC2 instance running Redis with 0 keyspace hits and is not containerized, a leader, a follower, nor a shard. | [Redis Integration][12] |
+| Unused resource | AWS | EC2 | Unused EC2 instance running memcached | EC2 instance running memcached with 0 keyspace hits and is not containerized. | [Memcache Integration][12] |
```
Comment (JackHFinkel) [inline content/en/cloud_cost_management/recommendations/_index.md:84]: memcached is not capitalized in docs

**Review by JackHFinkel — COMMENTED** [review]
[Doc readme](https://slack.com/help/articles/221769328-Locate-your-Slack-URL-or-ID) indicates I should use my slack username, but doesnt seem like that worked for me. I used jack.finkel instead as a guess. Maybe that should be updated to be more clear

**Comment by hestonhoffman** [comment]
> [Doc readme](https://github.com/DataDog/documentation?tab=readme-ov-file#datadog-staff) indicates I should use my slack username, but doesnt seem like that worked for me. I used jack.finkel instead as a guess. Maybe that should be updated to be more clear

I'll update the README. That link should go to https://dd.slack.com/account/settings#username

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**